src/recipes/avocado-juice.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "Avocado Juice",
+    "description": "The avocado, also known as butter pear or alligator pear, is a fruit that is widely acknowledged to have properties that reduce cholesterol levels in the body. To top it off, it's absolutely delicious. so this is avocado juice recipe",
+    "github": "misspuff55",
+    "ingredients": [
+        "5 Tablespoons sugar",
+        "5 Tablespoons water",
+        "2 avocados, peeled and pit removed",
+        "½ cup milk,
```

```suggestion
        "½ cup milk",
```

Also, can you move the images to the new location at `src/assets/recipes/`?

---

done sir, thanks btw